### PR TITLE
Update: Create user resource for navi WS

### LINKED ISSUE #193: User resource

### PR DESCRIPTION
**Pull request by cythrawll**
Had to mess around with elide settings so we could format the date per navi date standards.  Now that elide has sane date handling it makes it a bit easier, but custom formats is a bit awkward.

Also added logback so we can control logging.

More tests will be done on User endpoint as we add more resources.

Fixes #193

### CODE REVIEW

File: packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/User.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+@Entity
+@Table(name="navi_users")
+@Include(rootLevel = true, type = "users")
+@SharePermission
+@DeletePermission(expression = "nobody")
+@CreatePermission(expression = "is the same user")
+@UpdatePermission(expression = "is the same user now")
+class User {
+    @get:Id
```
[the remark this answers is not in the document]
Comment (cythrawll) [reply packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/User.kt]: I think you mean `@get:Size(min=1)` added it.

File: packages/webservice/pom.xml [hunk 2]
```diff
@@ -1,95 +1,128 @@
...
+            <groupId>com.jayway.restassured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <version>2.6.0</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.23.4</version>
```
[the remark this answers is not in the document]
Comment (cythrawll) [reply packages/webservice/pom.xml]: Fixed with gradle build

File: packages/webservice/app/src/test/kotlin/com/yahoo/navi/ws/test/integration/UserTest.kt [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+import com.yahoo.navi.ws.test.framework.IntegrationTest
+import com.yahoo.navi.ws.test.framework.matchers.RegexMatcher
+import org.apache.http.HttpStatus
+import org.junit.Test
+import org.hamcrest.Matchers.*
+import com.jayway.restassured.RestAssured.given
+
+
+/**
```
Comment (jkusa) [inline packages/webservice/app/src/test/kotlin/com/yahoo/navi/ws/test/integration/UserTest.kt]: Can we put this at the top?

File: packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/User.kt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
```
Comment (jkusa) [inline packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/User.kt]: Sorry, can we do `notBlank`?

**Pull request merged**